### ISSUE TITLE
a space anywhere in the path makes the project wont compile

### ISSUE THREAD

**Issue by jantje** [Sloeber/arduino-eclipse-plugin#52]
I had a space in my path and I got following error when compiling (verifying or building):

16:15:34 ***\* Incremental Build of configuration Release for project test ****
make all 
Building file: ../test.cpp
Starting C++ compile
"/home/jan/programs/arduino-1.5.2/hardware/tools/avr/bin/avr-g++" -c -g -Os -Wall -fno-exceptions -ffunction-sections -fdata-sections -MMD -mmcu=atmega328p -DF_CPU=16000000L -DARDUINO=152    -I"/home/jan/programs/arduino-1.5.2/hardware/arduino/avr/cores/arduino" -I"/home/jan/programs/arduino-1.5.2/hardware/arduino/avr/variants/standard" -MMD -MP -MF"test.cpp.d" -MT"test.cpp.d" -x c++ "../test.cpp"  -o  "test.cpp.o"
Finished building: ../test.cpp

make: **\* No rule to make target `/home/jan/workspaces/eclipse', needed by`test.elf'.  Stop.

16:15:35 Build Finished (took 645ms)

basically there was a space after "/home/jan/workspaces/eclipse" in "/home/jan/workspaces/eclipse plugin"

replacing the space with _ solved the problem. It took a while for the indexer to catch up.

Best regards
Jantje
